caddy/caddymain/run.go

```diff
@@ -34,6 +34,7 @@ func init() {
 	flag.StringVar(&cpu, "cpu", "100%", "CPU cap")
 	flag.BoolVar(&plugins, "plugins", false, "List installed plugins")
 	flag.StringVar(&caddytls.DefaultEmail, "email", "", "Default ACME CA account email address")
+	flag.IntVar(&caddytls.HTTPTimeout, "catimeout", 10, "ACME CA HTTP timeout in seconds")
```

Comment:
This should be a DurationVar. Then you don't have to convert it when we actually use it.